tests/geometry/test_data/offset_wire2022-04-08 10:19:27.json

```diff
@@ -1,8002 +1 @@
-                                ],
-                                [
-                                    9.583993802218922,
-                                    0.0,
-                                    -8.401214083657168
-                                ],
-                                [
-                                    9.558462727056996,
-                                    0.0,
-                                    -8.412360690626148
-                                ],
-                                [
-                                    9.532770019941939,
-                                    0.0,
-                                    -8.423172598321173
-                                ],
-                                [
-                                    9.506897081166784,
-                                    0.0,
-                                    -8.433650341502513
-                                ],
-                                [
-                                    9.480880350562396,
-                                    0.0,
-                                    -8.443775038537426
-                                ],
-                                [
-                                    9.454690978373451,
-                                    0.0,
-                                    -8.453551271660157
-                                ],
-                                [
-                                    9.402007102584124,
-                                    0.0,
-                                    -8.472379345552849
-                                ],
-                                [
-                                    9.375508635679713,
-                                    0.0,
-                                    -8.481432971066365
-                                ],
-                                [
-                                    9.348853364695518,
-                                    0.0,
-                                    -8.490119364793156
-                                ],
-                                [
-                                    9.322047638206858,
-                                    0.0,
-                                    -8.49843066151895
-                                ],
-                                [
-                                    9.295095994047294,
-                                    0.0,
-                                    -8.506359075326554
-                                ],
-                                [
-                                    9.268000182101819,
-                                    0.0,
-                                    -8.513896891342178
-                                ],
-                                [
-                                    9.240761528576597,
-                                    0.0,
-                                    -8.52103607309287
-                                ],
-         
```

Comment:
This file name is not valid on Windows. In fact, it causes an error if I try to clone the repository on my Windows machine. We should remove the time/colons

Reply:
In fact, this is on develop! Sorry I should have caught this before

Reply:
No worries, fix pushed